swan-cern-system/templates/fluentd/fluentd_sources.conf.yaml

```diff
@@ -12,9 +12,19 @@ data:
 
       {{ if not .Values.fluentd.output.includeInternal }}
       <match kubernetes.var.log.containers.**_kube-system_**>
+        {{ if not .Values.fluentd.output.includeFalco }}
           @type null
+        {{ else }}
+          <filter kubernetes.var.log.containers.**_kube-system_**>
```

Comment:
Maybe a comment would be necessary here? Something like keep the logs of the falco pod that outputs the audit messages.

Reply:
Still missing comment.